openwisp_radius/api/views.py

```diff
@@ -419,7 +419,12 @@ def post(self, request, *args, **kwargs):
 
 class DispatchOrgMixin(object):
     def dispatch(self, *args, **kwargs):
-        self.organization = get_object_or_404(Organization, slug=kwargs['slug'])
+        try:
+            self.organization = Organization.objects.select_related(
+                'radius_settings'
+            ).get(slug=kwargs['slug'])
+        except Organization.DoesNotExist:
```

Comment:
If we use the [DRF NotFound exception](https://www.django-rest-framework.org/api-guide/exceptions/#notfound), the API response should be more consistent with the rest of the responses from the API. Can you try it and let me know?

Reply:
For some reason, this is not being handled by the DRF exception handler. Look at the screenshot attached below, this is what I get.

![image](https://user-images.githubusercontent.com/10897423/101132304-ca92a080-362c-11eb-97a6-d3c95e9a74d4.png)